core/src/main/resources/jenkins/formelementpath/form-element-path.js

```diff
@@ -0,0 +1,229 @@
+(function () {
+    function pack(n, v) {
+        var o = {};
+        o[n] = v;
+        return o;
+    }
+
+    function pre(selector, behavior) {
+        if (Behaviour.specify) {
+            Behaviour.specify(selector, "form-element-path", -200, behavior);
+        } else {
+            Behaviour.list.unshift(pack(selector, behavior));
+        }
+    }
+
+    function post(selector, behavior) {
+        if (Behaviour.specify) {
+            Behaviour.specify(selector, "form-element-path", 200, behavior);
+        } else {
+            Behaviour.list.shift(pack(selector, behavior));
+        }
+    }
+
+
+    // patch hetero-list-container handling to capture name before hudson-behavior.js kicks in
+    // in Jenkins 1.473 and newer the suffix is already set in hetero-list.js
```

Comment:
Probably an opportunity to clean up here.

Reply:
Possibly yes I saw this but didn’t quite understand it, I’ll give it a bit of a test.

FTR I did remove some more obvious cruft from old versions